custom-modifiers/waitforelementtobevisiblemodifier/pom.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is pretty cool stuff here, IMO generic enough to move it to [`core modifiers`](https://github.com/Cognifide/aet/tree/master/core/jobs/src/main/java/com/cognifide/aet/job/common/modifiers) e.g. under `waitfor.elementtobevisible` and `waitfor.imageiscomplete` packages.
Don't forget then about adding Apache License headers.